app/__init__.py

```diff
@@ -17,7 +17,7 @@
 from werkzeug.utils import import_string
 from .database import db
 from app.utils import get_s3_cdn_prefix
-from app.auth.controllers import auth_blueprint
+from app.auth.controllers import auth_blueprint, datastore_blueprint
```

Comment:
Why is it called `datastore_blueprint`. As discussed, this is really about the `bitstore`. We are only using `datastore` in the API to maintain compatibility with OS as it is right now!